west.yml

```diff
@@ -114,7 +114,7 @@ manifest:
       revision: 07f430dac6c400abdd358ba1173c0f94b94a5a80
       path: modules/lib/openthread
     - name: segger
-      revision: 874d9e9696b00c09f9eeefe839028dc25fe44983
+      revision: f5226b01cb630f706db190b1573fd0b04b6c2e95
```

Comment:
i think this commit is outdated and has some code that is not compatible with Zephyr anymore. the commit you need is already included in 874d9e9696b00c09f9eeefe839028dc25fe44983, so this change can be removed.

Reply:
I didn't notice, I have removed this change from the commit.